pkg/util/utils.go

```diff
@@ -69,3 +70,23 @@ func CommaStringToStringMap(s string) (map[string]string, error) {
 
 	return kvMap, nil
 }
+
+// EndPointHavePort will check for the presence of the port in the given endpoint
+// It will return true and nil, if the endpoint is valid and port is present
+// It will return false and nil, if the endpoint is valid and port is missing
+// It will return false and error, if the endpoint itself is not valid.
+func EndpointHavePort(endpoint string) (bool, error) {
```

Comment:
I think to make the code in `cmd/..` simpler have this function return the endpoint with the appropriate port. In other words, have it return `(string, error)` instead, where the string is the cleaned up, correct endpoint. You may need to rename the func and move the default const port here. You can also make the default port exported in case others want access.

Reply:
Fixed it.